guides/installation_guides/java/windows/README.md

```diff
@@ -1,5 +1,27 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Java Installation Guide
+### Download
+Download the Windows version of Java from the [Java website](https://www.java.com/en/download/) .
+
+### Install
+*Note: administrative access may be required for installation*
+
+1. Run the Java setup file (exe).
+2. Give installation access to the Java installer.
+3. Now give installation path or directory.
+4. Click on Install button.
```

Comment:
Lowercase I in install.

Reply:
The install button has I in upper case so should I change it ??